<launch>
    <arg name="config_path_left" default = "$(find feature_tracker)/../config/sportsfield/stereo_left_config.yaml" />
    <arg name="config_path_right" default = "$(find feature_tracker)/../config/sportsfield/stereo_right_config.yaml" />
    
    <node name="stereo_tracker" pkg="feature_tracker" type="stereo_tracker" output="screen">
        <param name="config_file_left" type="string" value="$(arg config_path_left)" />
        <param name="config_file_right" type="string" value="$(arg config_path_right)" />
    </node>


</launch>
